public class FunctionUse16178 {

    public void funcUse() {
		org.apache.hadoop.ha.FailoverFailedException failoverfailedexception = new org.apache.hadoop.ha.FailoverFailedException();
    }
}